--- a/internal-complibs/snappy-1.1.1/snappy.h
+++ b/internal-complibs/snappy-1.1.1/snappy.h
@@ -44,6 +44,14 @@

 #include "snappy-stubs-public.h"

+// Windows does not define ssize_t by default.  This is a workaround.
+// Please note that this is only defined in the Blosc sources of Snappy.
+#if defined(_WIN32) && !defined(__MINGW32__)
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
+#endif
+
+
 namespace snappy {
   class Source;
   class Sink;
